src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -615,7 +615,8 @@ private void AddRestoreBar()
         {
             if (Model.Context.PackageRestoreManager != null)
             {
-                RestoreBar = new PackageRestoreBar(Model.Context.SolutionManagerService, Model.Context.PackageRestoreManager);
+                var projectCotextInfo = Model.Context.Projects.Count() == 1 ? Model.Context.Projects.FirstOrDefault() : null;
```

Comment:
Could project count be more than 1?

Reply:
: nit `projectCotextInfo` -> `projectContextInfo`

Reply:
We are targeting this banner for project level only